packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/components/layout.jsx.hbs

```diff
@@ -0,0 +1 @@
+{{> nextDrupalLayout }}
```

Comment:
```suggestion
{{> nextDrupalLayout}}
```
We haven't really discussed our handlebars style guide, but imo the closing tag should not have a space before it. Open to other ways of doing it though.